migrations/20210311092126_doctor-patient-info.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable func-names */
+const dbPatients = require('../db/patients')
+
+exports.up = function(knex) {
+  return knex.schema.table(dbPatients.patientsTable, function (table) {
+    table.text('doctorName').notNullable()
```

Comment:
meme question ici, est-ce qu'on rend le champ optionnel... Je serai pour optionnel, pour etre moins contraignant.

Reply:
On en a discuter avec Inès sur Slack mardi qui confirme qu'un étudiant a forcément vu un médecin et qu'il sera en mesure de donner le nom de son médecin. Au besoin on le repassera en optionnel si cela pose problème trop souvent.

Reply:
je passe le champs en nullable dans la DB au cas où :slightly_smiling_face: et on force pour le moment le nom du medecin